sql/attrib_entries.sql

```diff
@@ -321,15 +321,20 @@ INSERT IGNORE INTO attrib_type (attrib_type_id, code, name, description) VALUES
 INSERT IGNORE INTO attrib_type (attrib_type_id, code, name, description) VALUES ( 532,'dbnsfp_revel_pred','dbNSFP REVEL prediction','dbNSFP REVEL prediction');
 INSERT IGNORE INTO attrib_type (attrib_type_id, code, name, description) VALUES ( 533,'dbnsfp_meta_lr_pred','dbNSFP MetaLR prediction','dbNSFP MetaLR prediction');
 INSERT IGNORE INTO attrib_type (attrib_type_id, code, name, description) VALUES ( 534,'dbnsfp_ma_pred','dbNSFP mutation assessor prediction','dbNSFP mutation assessor prediction');
-INSERT IGNORE INTO attrib_type (attrib_type_id, code, name, description) VALUES  (535, 'MANE_Select', 'MANE Select v0.7', 'MANE Select (v0.7) is the preliminary release (phase 3) of the MANE Select data set. The Matched Annotation from NCBI and EMBL-EBI project (MANE) is a collaboration between Ensembl-GENCODE and RefSeq to select a default transcript per human protein coding locus that is representative of biology, well-supported, expressed and conserved. This transcript set matches GRCh38 and is 100% identical between RefSeq and Ensembl-GENCODE for 5\' UTR, CDS, splicing and 3\' UTR.');
+INSERT IGNORE INTO attrib_type (attrib_type_id, code, name, description) VALUES ( 535,'MANE_Select','MANE Select v0.9','MANE Select (v0.9) is the preliminary release (phase 5) of the MANE Select data set. The Matched Annotation from NCBI and EMBL-EBI project (MANE) is a collaboration between Ensembl-GENCODE and RefSeq to select a default transcript per human protein coding locus that is representative of biology, well-supported, expressed and conserved. This transcript set matches GRCh38 and is 100% identical between RefSeq and Ensembl-GENCODE for 5\' UTR, CDS, splicing and 3\' UTR.');
```

Comment:
Can the spacing for 535 be kept the same - easier to see  changes?

Reply:
I see your point, however the spacing was conforming with other lines in the file. e.g lines above.

Reply:
I was asking if the spacing for 535 be kept as it was so that change/update are easier to see i.e. just a version update. 535 was conforming with the other lines in the file below.